newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -127,17 +133,21 @@ const ExtensionInstallDialog = ({
               !isCompatible ? (
                 <Trans>Not compatible</Trans>
               ) : alreadyInstalled ? (
-                extensionUpdate ? (
-                  extensionShortHeader.tier === 'community' ? (
-                    <Trans>Update (could break the project)</Trans>
-                  ) : (
-                    <Trans>Update</Trans>
-                  )
-                ) : (
-                  <Trans>Re-install</Trans>
-                )
-              ) : (
-                <Trans>Install in project</Trans>
+				fromStore ? (
+					extensionUpdate ? (
+					  extensionShortHeader.tier === 'community' ? (
+						<Trans>Update (could break the project)</Trans>
+					  ) : (
+						<Trans>Update</Trans>
+					  )
+					) : (
+					  <Trans>Re-install</Trans>
+					)
+				) : (
+					<Trans>Override</Trans>
```

Comment:
We could go for a simple worder I think: "Replace existing extension"?

Reply:
Thanks, that's right, i didn't use format, and ok for the Override change !